tatus

```diff
@@ -0,0 +1,89 @@
+[33mcommit c38820f1e77a830144bd2211dc64f1dca11aeb5e[m[33m ([m[1;36mHEAD -> [m[1;32mmaster[m[33m, [m[1;31morigin/master[m[33m, [m[1;31morigin/HEAD[m[33m)[m
```

Comment:
don't play with .git files
and modify commits 
it doesn't work that way

Reply:
@arpitBhalla you can do rebase if you want to merge, change order, change message of commits.